database/migrations/2017_04_28_151956_new_database_schema.php

```diff
@@ -0,0 +1,304 @@
+<?php
+
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class NewDatabaseSchema extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('global_options', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('name'); //TODO remove
+            $table->string('code'); //TODO rename to key
+            $table->string('value');
+            $table->text('description')->nullable();
+            $table->integer('not_editable')->nullable();
+            $table->timestamps();
+        });
+
+        Schema::create('seeder_logs', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('code'); //TODO rename to key
+            $table->timestamps();
+        });
+
+        Schema::create('sessions', function (Blueprint $table) {
```

Comment:
Sessions table is autogenerated via laravel. I doubt you should modify anything from it :)